public/app/features/api-keys/ApiKeysTable.tsx

```diff
@@ -25,10 +29,19 @@ export const ApiKeysTable: FC<Props> = ({ apiKeys, timeZone, onDelete }) => {
         <tbody>
           {apiKeys.map((key) => {
```

Comment:
❓ do you know if the API keys are sorted using time? I am thinking that removing the toggle can lead to the display of a ton of expired keys together with valid ones, making it a bit hard to see which ones are valid, also cleaning them is going to be quite painful (clicking delete one by one)

Reply:
i believe they're sorted alphabetically currently